SimCalorimetry/HcalTrigPrimAlgos/src/HcalTriggerPrimitiveAlgo.cc

```diff
@@ -858,7 +858,8 @@ void HcalTriggerPrimitiveAlgo::setWeightsQIE11(const edm::ParameterSet& weightsQ
   // Names are just abs(ieta) for HBHE
   std::vector<std::string> ietaStrs = weightsQIE11.getParameterNames();
   for (auto& ietaStr : ietaStrs) {
-    weightsQIE11_[std::stoi(ietaStr)] = weightsQIE11.getUntrackedParameter<std::vector<double>>(ietaStr);
+    // Strip off "ieta" part of key and just use integer value in map
+    weightsQIE11_[std::stoi(ietaStr.substr(4))] = weightsQIE11.getUntrackedParameter<std::vector<double>>(ietaStr);
```

Comment:
this will then become
```
auto const & v = weightsQIE11.getUntrackedParameter<std::vector<double>>(ietaStr);
weightsQIE11_[std::stoi(ietaStr.substr(4))-1] = {v[0],v[1]};
```

btw WHY "Untracked"? are These parameters not changing the output of the simulation?

Reply:
fi you want to keep the addressing between 1-28, just make it 29-long and waste 0...

Reply:
I removed the last bit of "untracked" from this. Also, I decided to make the array 29 long to avoid any `-1`.